Symfony/CS/Tests/Fixer/Symfony/SingleLineFunctionDeclarationFixerTest.php

```diff
@@ -0,0 +1,310 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests\Fixer\Symfony;
+
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+
+/**
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ */
+final class SingleLineFunctionDeclarationFixerTest extends AbstractFixerTestBase
+{
+    /**
+     * @dataProvider provideFixCases
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->makeTest($expected, $input);
+    }
+
+    public function provideFixCases()
+    {
+        return array(
+            array(
+                '<?php
+                class Foo extends Bar implement BarInterface, \Traversable
+                {
+                    function foo(Bar $thing, $other)
+                    {
+                    }
+                }',
+            ),
+            array(
+                '<?php
+                function foo(Bar $thing, $other)
+                {
+                }',
+            ),
+            array(
+                '<?php
+    /**
+     * Returns the name of the configuration.
+     *
+     * The name must be all lowercase and without any spaces.
+     *
+     * @return string The name of the configuration
+     */
+    public function getName();
+
+    /**
+     * Returns the description of the configuration.
+     *
+     * A short one-line description for the configuration.
+     *
+     * @return string The description of the configuration
```

Comment:
Yes. Because when I run the fixer on the code, it fucked up all your comments! x)